charts/k8s-service/README.md

```diff
@@ -48,6 +49,13 @@ The following resources will be deployed with this Helm Chart, depending on whic
 - `ManagedCertificate`: The `ManagedCertificate` is a [GCP](https://cloud.google.com/) -specific resource that creates a Google Managed SSL certificate. Google-managed SSL certificates are provisioned, renewed, and managed for your domain names. Read more about Google-managed SSL certificates [here](https://cloud.google.com/load-balancing/docs/ssl-certificates#managed-certs). Created only if you configure the `google.managedCertificate` input (and set
                          `google.managedCertificate.enabled = true` and `google.managedCertificate.domainName = your.domain.name`).
 
+back to [root README](/README.adoc)
+
+## How do I deploy additional services not managed by the chart?
+
+You can create custom service profiles within the `customResources` key in the chart. You provide the service profile manifest directly as a value under `customResources.resources` and set `customResources.enabled` to `true`. For examples of custom resources, take a look at the examples in [test/fixtures/custom_resources_values.yaml](../../test/fixtures/custom_resources_values.yaml) and [test/fixtures/multiple_custom_resources_values.yaml](../../test/fixtures/multiple_custom_resources_values.yaml).
```

Comment:
NIT:

```suggestion
You can create custom Kubernetes resources that are not directly managed by the chart within the `customResources` key in the chart. You provide the resource manifests directly as a value under `customResources.resources` and set `customResources.enabled` to `true`. For examples of custom resources, take a look at the examples in [test/fixtures/custom_resources_values.yaml](../../test/fixtures/custom_resources_values.yaml) and [test/fixtures/multiple_custom_resources_values.yaml](../../test/fixtures/multiple_custom_resources_values.yaml).
```

Reply:
Thank you! Wondered about the wording here.